File: TUKulinarium_API/Data/Models/Pizza.cs
using TUKulinarium_API.Helpers.BaseClasses;
using TUKulinarium_API.Helpers.Enums;

namespace TUKulinarium_API.Data.Models
{
    public class Pizza : Dish
    {
        public string Ingredients {get; set;}
        public Sauces PizzaSauce {get; set;}
        public DoughTypes PizzaDough {get; set;}
        public PizzaSizes PizzaSize {get; set;}
    }
}

File: TUKulinariumClient/src/app/models/Pizza.ts
import { DoughType } from '../enums/DoughTypes';
import { Sauces } from '../enums/Sauces';
import { PizzaSizes } from '../enums/Sizes';
import { Dish } from './Dish';

export class Pizza extends Dish {
  pizzaSauce: Sauces;
  pizzaDough: DoughType;
  pizzaSize: PizzaSizes;

  
}
